### ISSUE TITLE
template URL for example todo app incorrectly set

### ISSUE THREAD

**Issue by tkothe** [slushjs/slush-angular#51]
I just generated my first app (let's call it ``myApp``) with slush-angular. I was surprised I did not see the todo app that the generator asked me for. 
Here's how the app.js looked like after the slush generator finished:

```javascript
angular.module('myApp', [
  'ngRoute',
  'myApp.todo'
])
.config(function ($routeProvider) {
  'use strict';
  $routeProvider
    .when('/todo', {
      controller: 'TodoCtrl',
      templateUrl: '/myApp/todo/todo.html'
    })
    .otherwise({
      redirectTo: '/todo'
    });
});
``` 

and here's what I had to change it to, in order to get the todo app on the screen (hint: I had to remove ``myApp`` from the ``templateUrl``)

```javascript
angular.module('myApp', [
  'ngRoute',
  'myApp.todo'
])
.config(function ($routeProvider) {
  'use strict';
  $routeProvider
    .when('/todo', {
      controller: 'TodoCtrl',
      templateUrl: '/todo/todo.html'
    })
    .otherwise({
      redirectTo: '/todo'
    });
});
``` 
After that it worked as expected.

I currently don't have the time to dig into slush-angular in order to fix this myself, but this should really only be a minor issue.

**Comment by tkothe** [comment]
It might be as easy as removing ``/<%= nameDashed %>`` [here](https://github.com/slushjs/slush-angular/blob/master/templates/src/app/app.js#L11). But someone who knows something about this should double check :smile:

**Comment by joakimbeng** [comment]
Fixed by #50

**Comment by tkothe** [comment]
ooops - didn't see that :smile:  thanks!
